### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by Soreine**
https://soreine.github.io/hyper-gwent/

This PR has got #32  #31  #30  packed in

screen shot 2017-08-06 at 14 48 05

### CODE REVIEW

**Comment by Soreine** [comment]
Missing a favicon (the icon of the extension)

**Comment by Soreine** [comment]
Close #11

File: manifest.json [hunk 1]
```diff
@@ -1,19 +1,22 @@
 {
+  "manifest_version": 2,
+  "author": "Soreine & Zhouzi",
+  "homepage": "https://soreine.github.io/hyper-gwent",
   "name": "Hyper Gwent",
-  "description": "Enrich pages containing GwentⓇ related content. Card names quick access, terms highlight and more!",
+  "description": "Show tooltips for cards in pages containing GwentⓇ related content.",
```
Comment (Zhouzi) [inline manifest.json]: Not sure if that's an issue but this description is different than the one on the website.

File: manifest.json [hunk 2]
```diff
@@ -1,19 +1,22 @@
 {
+  "manifest_version": 2,
+  "author": "Soreine & Zhouzi",
```
Comment (Zhouzi) [inline manifest.json]: ❤️

File: src/website/index.js [hunk 3]
```diff
@@ -2,31 +2,95 @@
...
+      </div>
+
+    </div>
+
+    <a className="download-link" href={EXT_LINK}>{'Add to Chrome'}</a>
+
+    <p className="issues">
+      If you find bugs, if you want to submit new acronyms, request a feature, or contribute, post an issue <a href={REPO}>here</a>.
+      <br/>
+      You can also contact <a href="https://www.reddit.com/message/compose/?to=Soreine">/u/Soreine</a>.</p>
```
Comment (Zhouzi) [inline src/website/index.js]: We could customize the HTML template to include all this, it'd be better for SEO & all. The Webpack loader would just insert the JavaScript specific code.

**Pull request merged**